interface/modules/custom_modules/oe-module-faxsms/table.sql

```diff
@@ -0,0 +1,13 @@
+SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
+START TRANSACTION;
```

Comment:
CREATE TABLE as a SQL DDL will automatically close any outstanding transactions so you don't need to bundle this in a transaction.